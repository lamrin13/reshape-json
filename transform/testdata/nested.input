{
    "name": "John",
    "street": "Dongg Street",
    "unit": "12",
    "role": "SWE",
    "last4": "1234",
    "expiryMonth": 12,
    "expiryYear": 2023,
    "fruits": ["Apple", "Banana", "Orange"]
}