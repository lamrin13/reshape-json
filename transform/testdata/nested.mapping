{
    "user.name":          "name",
    "user.role":          "role",
    "card.lastFour":      "last4",
    "card.expiry.Month":  "expiryMonth",
    "card.expiry.Year":   "expiryYear",
    "address":            "unit+street/-",
    "user.favoriteFoods": "fruits"
}