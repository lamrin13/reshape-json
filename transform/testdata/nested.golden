{
    "user": {
        "name": "John",
        "role": "SWE",
        "favoriteFoods": ["Apple", "Banana", "Orange"]
    },
    "address": "12-Dongg Street",
    "card": {
        "lastFour": "1234",
        "expiry": {
            "Month": 12,
            "Year": 2023
        }
    }
}